requirements-dev.txt

```
@@ -9,7 +9,7 @@ pep8-naming
 pycodestyle
 pyflakes
 black
```

```suggestion
black<25
```
Try this to fix the black version. It's complaining about needing to reformat code that isn't in this proposed bundle and we don't want to modify that right now.